packages/manifold/contracts/gachaclaims/ERC1155GachaLazyClaim.sol

```diff
@@ -0,0 +1,256 @@
+      interfaceId == type(IGachaLazyClaim).interfaceId ||
+      interfaceId == type(ICreatorExtensionTokenURI).interfaceId ||
+      interfaceId == type(IAdminControl).interfaceId ||
+      interfaceId == type(IERC165).interfaceId;
+  }
+
+  constructor(address initialOwner) GachaLazyClaim(initialOwner) {}
+
+  /**
+   * See {IERC1155GachaLazyClaim-initializeClaim}.
+   */
+  function initializeClaim(
+    address creatorContractAddress,
+    uint256 instanceId,
+    ClaimParameters calldata claimParameters
+  ) external payable override adminRequired {
+    if (instanceId == 0 || instanceId > MAX_UINT_56) revert IGachaLazyClaim.InvalidInstance();
+    if (_claims[creatorContractAddress][instanceId].storageProtocol != StorageProtocol.INVALID)
+      revert IGachaLazyClaim.ClaimAlreadyInitialized();
+    // Checks
+    if (claimParameters.storageProtocol == StorageProtocol.INVALID) revert IGachaLazyClaim.InvalidStorageProtocol();
+    if (claimParameters.endDate != 0 && claimParameters.startDate >= claimParameters.endDate)
+      revert IGachaLazyClaim.InvalidDate();
+
+    address[] memory receivers = new address[](1);
+    receivers[0] = msg.sender;
+    uint256[] memory amounts = new uint256[](claimParameters.tokenVariations);
+    string[] memory uris = new string[](claimParameters.tokenVariations);
+    uint256[] memory newTokenIds = IERC1155CreatorCore(creatorContractAddress).mintExtensionNew(receivers, amounts, uris);
+
+    if (newTokenIds[0] > MAX_UINT_80) revert IGachaLazyClaim.InvalidStartingTokenId();
+
+    // Create the claim
+    _claims[creatorContractAddress][instanceId] = Claim({
+      storageProtocol: claimParameters.storageProtocol,
+      total: 0,
+      totalMax: claimParameters.totalMax,
+      startDate: claimParameters.startDate,
+      endDate: claimParameters.endDate,
+      startingTokenId: uint80(newTokenIds[0]),
+      tokenVariations: claimParameters.tokenVariations,
+      location: claimParameters.location,
+      paymentReceiver: claimParameters.paymentReceiver,
+      cost: claimParameters.cost,
+      erc20: claimParameters.erc20
+    });
+    for (uint256 i; i < claimParameters.tokenVariations; ) {
+      _tokenInstances[creatorContractAddress][newTokenIds[i]] = instanceId;
+      unchecked {
+        ++i;
+      }
+    }
+
+    emit GachaClaimInitialized(creatorContractAddress, instanceId, msg.sender);
+  }
+
+  /**
+   * See {IERC1155GachaLazyClaim-updateClaim}.
+   */
+  function updateClaim(
+    address creatorContractAddress,
+    uint256 instanceId,
+    ClaimParameters memory claimParameters
+  ) external override adminRequired {
+    Claim memory claim = _getClaim(creatorContractAddress, instanceId);
+    if (claimParameters.endDate != 0 && claimParameters.startDate >= claimParameters.endDate)
+      revert IGachaLazyClaim.InvalidDate();
+    if (claimParameters.totalMax < claim.totalMax) revert IGachaLazyClaim.CannotLowerTotalMaxBeyondTotal();
```

Comment:
Wrong check.  They CAN lower the totalMax, they just can't lower it below the total minted so far.